engine/install/index.md

```diff
@@ -46,23 +46,23 @@ your preferred operating system below.
 
 {% assign yes = '![yes](/images/green-check.svg){: .inline style="height: 14px; margin: 0 auto"}' %}
 
-| Platform                                                          | x86_64 / amd64                                   |
-|:------------------------------------------------------------------|:------------------------------------------------:|
-| [Docker Desktop for Mac (macOS)](../../docker-for-mac/install.md) | [{{ yes }}](../../docker-for-mac/install.md)     |
-| [Docker Desktop for Windows](../../docker-for-windows/install.md) | [{{ yes }}](../../docker-for-windows/install.md) |
+| Platform                                                          | x86_64 / amd64                                   | Apple Silicon                                    |
```

Comment:
Thanks @thaJeztah. Suggest we use ARM64 here instead of Apple silicon. LGTM otherwise

![image](https://user-images.githubusercontent.com/47779042/124573212-3d712d80-de41-11eb-9399-0065739e2e0a.png)

Reply:
Yeah, I was in doubt what to use. We have enough space; perhaps both? (`ARM64 (Apple Silicon)`) WDYT?

Reply:
Yes, that sounds good and eliminates any confusion.

Reply:
Updated 👍 

Also changed the `ARM` architectures to lowercase (looking at them again, they felt a bit "shouty" and it wasn't consistent with `x86_63 / amd64`)